scenedetect.cfg

```diff
@@ -132,10 +132,8 @@
 
 # Threshold value (float) that the calculated difference between subsequent
 # histograms must exceed to trigger a new scene.
-#threshold = 20000.0
+#threshold = 0.95
```

Comment:
You'll also need to update the ranges and config options specified for the CLI here:
https://github.com/Breakthrough/PySceneDetect/blob/develop/scenedetect/_cli/config.py#L281

Reply:
I can help out with this if need be, and this isn't strictly necessary if there are follow up changes you wish to make.  If that is the case though, we should remove the test cases that try to use this from the CLI, until this is stable.